DogTales/src/player.cpp

```diff
@@ -21,8 +21,12 @@ void Player::handle_wall_collision() {
 	auto const bounce_rect = bave::Rect<>::from_size(m_world_space - m_sprite.get_size(), glm::vec2{0.0f});
 
 	// if the sprite's position exceeds the play area, the corresponding velocity component needs to flip.
-	if (position.x < bounce_rect.top_left().x || position.x > bounce_rect.bottom_right().x) { m_physics.velocity.x *= -0.9f; }
```

Comment:
Sometimes I really prefer this compact code style. Not sure if there's a way to allow for it in the formatting check, or perhaps you disagree with using the compact style, in which case, I'm curious to know why!

Reply:
Yeah me too, and it's already set to format that way! Unless the line would be too long, which here exceeds the 120 column limit.